src/SFA.DAS.Admin.Roatp.Web/Models/OfferApprenticeshipsViewModel.cs

```diff
@@ -2,8 +2,12 @@
 
 namespace SFA.DAS.Admin.Roatp.Web.Models;
 
-public class ApprenticeshipsUpdateViewModel : IBackLink
+public class OfferApprenticeshipsViewModel : OfferApprenticeshipsSubmitModel, IBackLink
 {
     public List<ApprenticeshipsSelectionModel> ApprenticeshipsSelection { get; set; } = new();
+}
+
+public class OfferApprenticeshipsSubmitModel
+{
     public bool? ApprenticeshipsSelectionChoice { get; set; }
```

Comment:
Since this is a boolean, the variable name should be a question ie. IsApprenticeshipsOffered